src/types/index.ts

```diff
@@ -166,6 +166,13 @@ export interface SelectProps {
      *  @category Common
      */
     flatListProps?: Omit<FlatListProps<OptionType>, 'data' | 'renderItem' | 'ListEmptyComponent'>;
+    /**
+     *  if `true` then multi option can be picked
+     *
+     *  @category Common
+     *  @default false
+     */
+    multiSelection?: boolean;
```

Comment:
also add this prop to `website/docs/api/common.md`

Reply:
👍